### ISSUE TITLE
Add new notations module for science

### ISSUE THREAD

**Issue by CoolPlayLin** [faker-js/faker#2416]
### Clear and concise description of the problem

Add notations module to science so that we could use symbol to express units in science

### Suggested solution

According to https://en.wikipedia.org/wiki/List_of_common_physics_notations

**Comment by ST-DDT** [comment]
What is the difference between this and [faker.science.unit](https://fakerjs.dev/api/science.html#unit)?

**Comment by CoolPlayLin** [comment]
In science, unit is used to mean possessed by a physical quantity. notations is used to mean code of physical unit.
For example, The formula of pressure is `P = F / S`. The P means *pressure*, F means *force* and S means *square*. Both they are notations.

**Comment by CoolPlayLin** [comment]
@ST-DDT 
In science, unit is used to mean possessed by a physical quantity. notations is used to mean code of physical unit.
For example, The formula of pressure is `P = F / S`. The P means *pressure*, F means *force* and S means *square*. Both they are notations instead of units.
Anyway, the unit of F is Newton, P is Pascal, S can use meter, centmeter, kilometer and etc.

**Comment by matthewmayer** [comment]
i would call things like force, pressure, area "quantities" https://en.wikipedia.org/wiki/Physical_quantity - the notation is just the symbol.

**Comment by CoolPlayLin** [comment]
```typescript
export interface Notation {
  /**
   * The long version of the notations (e.g. `work`).
   */
  name: string;
  /**
   * The short version/abbreviation of the notations (e.g. `w`)
   */
  symbol: string;
}
```

However, one notation can refer to more than one meaning.( e.g. uppercase C can refer [capacitance](https://en.wikipedia.org/wiki/Capacitance), [heat capacity](https://en.wikipedia.org/wiki/Heat_capacity) and [constant of integration](https://en.wikipedia.org/wiki/Constant_of_integration) )
I don't think this interface is good. Could you tell me how I modify it?😊

@matthewmayer

**Comment by CoolPlayLin** [comment]
Maybe a good way
```ts
export interface Notation {
  /**
   * The long version of the notations (e.g. `work`).
   */
  name: string;
  /**
   * The short version/abbreviation of the notations (e.g. `w`)
   */
  symbol: string[];
}
```

**Comment by matthewmayer** [comment]
I would wait for more upvotes. One of the reason to wait for 10+ upvotes is to get a variety of opinions over what use cases people might need this for. That informs making sensible naming decisions. 

I don't use the science module in any of my projects, but if we can find 10 people who do over a few months, they'd have a much better idea of how to structure the api.

**Comment by ejcheng** [comment]
> i would call things like force, pressure, area "quantities" [en.wikipedia.org/wiki/Physical_quantity](https://en.wikipedia.org/wiki/Physical_quantity) - the notation is just the symbol.

I agree with this--- "notation" is too similar to "unit", so if we implement it, we should name this method "quantity".
